update doc related to fast-create (#18517)
pingcap/docs: accelerated-table-creation.md

TiDB v7.6.0 introduces the system variable [`tidb_ddl_version`](https://docs.pingcap.com/tidb/v7.6/system-variables#tidb_enable_fast_create_table-new-in-v800) to support accelerating table creation, which improves the efficiency of bulk table creation. Starting from v8.0.0, this system variable is renamed to [`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-new-in-v800).

> When accelerated table creation is enabled, table creation statements for the same schema submitted to the same TiDB node at the same time will be merged into a batch table creation statement to improve table creation performance. Therefore, to improve the table creation performance, you need to connect to the same TiDB node to create tables under the same schema concurrently and increase the concurrency appropriately.

Reviewer: ```suggestion
When ‘tidb_enable_fast_create_table’ is enabled, table creation statements for the same schema submitted to the same TiDB node at the same time will be merged into a batch table creation statement to improve table creation performance. Therefore, to improve the table creation performance, you need several sessions to connect to the same TiDB node to create tables under the same schema concurrently and increase the concurrency appropriately. 
```

> When [`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-new-in-v800) is enabled, table creation statements for the same schema submitted to the same TiDB node at the same time will be merged into a batch table creation statement to improve table creation performance. Therefore, to improve the table creation performance, you need several sessions to connect to the same TiDB node to create tables under the same schema concurrently and increase the concurrency appropriately.

Reviewer 2: ```suggestion
When you enable accelerated table creation via [`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-new-in-v800), table creation statements with the same schema committed to the same TiDB node at the same time are merged into batch table creation statements to improve table creation performance. Therefore, to improve the table creation performance, try to connect to the same TiDB node, create tables with the same schema concurrently, and increase the concurrency appropriately.
```

> The merged bulk table creation statements are executed within the same transaction, so if one of them fails, all of them will fail.

Reviewer 2: ```suggestion
The merged batch table creation statements are executed within the same transaction, so if one statement of them fails, all of them will fail.
```
